### PR TITLE
Change site url each language.

### PR DESCRIPTION
**Pull request by operando**
Change site url each language.

## Question

**Do you have Arabic language site?**

Sorry, It's a joke.

In the case Arabic language, displayed the english site.

## Japanese

device-2016-02-11-222504

### English

device-2016-02-11-222442

## Arabic

device-2016-02-11-223813

### CODE REVIEW

**Comment by konifar** [comment]
Thanks!

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -61,6 +61,7 @@
     <string name="about_license">License</string>
     <string name="about_questionnaire">Questionnaire</string>
     <string name="about_version_prefix">"V%1$s</string>
+    <string name="about_site_url">https://droidkaigi.github.io/2016/en</string>
```
Comment (konifar) [inline app/src/main/res/values/strings.xml]: One suggestion.
I guess it is better to create `values-ar/strings_urls.xml` and `values-ja/strings_urls.xml` and `write about_site_url` in that new file. How do you think?

Comment (operando) [inline app/src/main/res/values/strings.xml]: Good suggestion!!
Fix committed.

File: app/src/main/java/io/github/droidkaigi/confsched/fragment/AboutFragment.java [hunk 2]
```diff
@@ -54,8 +55,8 @@ private void initView() {
         binding.txtRep.setText(repText);
         AppUtil.linkify(getActivity(), binding.txtRep, REP_TWITTER_NAME, AppUtil.getTwitterUrl(REP_TWITTER_NAME));
 
-        binding.txtSiteUrl.setText(LocaleUtil.getRtlConsideredText(SITE_URL, getContext()));
-        AppUtil.linkify(getActivity(), binding.txtSiteUrl, SITE_URL, SITE_URL);
+        binding.txtSiteUrl.setText(LocaleUtil.getRtlConsideredText(siteUrl, getContext()));
+        AppUtil.linkify(getActivity(), binding.txtSiteUrl, siteUrl, siteUrl);
```
Comment (konifar) [inline app/src/main/java/io/github/droidkaigi/confsched/fragment/AboutFragment.java]: One small suggestion.
I guess `siteUrl` should be local variable.

```java
String siteUrl = getString(R.string.about_site_url);
binding.txtSiteUrl.setText(LocaleUtil.getRtlConsideredText(siteUrl, getContext()));
AppUtil.linkify(getActivity(), binding.txtSiteUrl, siteUrl, siteUrl);
```

Comment (operando) [inline app/src/main/java/io/github/droidkaigi/confsched/fragment/AboutFragment.java]: Wow!! Thanks suggestion.
Changed `siteUrl` local variable.

**Comment by konifar** [comment]
LGTM!

Well done sir!

**Pull request merged**